ddtrace/settings/_config.py

```diff
@@ -454,18 +448,8 @@ def __init__(self):
 
         self._span_traceback_max_size = _get_config("DD_TRACE_SPAN_TRACEBACK_MAX_SIZE", 30, int)
 
-        # Master switch for turning on and off trace search by default
-        # this weird invocation of getenv is meant to read the DD_ANALYTICS_ENABLED
-        # legacy environment variable. It should be removed in the future
-        self._analytics_enabled = _get_config(["DD_TRACE_ANALYTICS_ENABLED", "DD_ANALYTICS_ENABLED"], False, asbool)
-        if self._analytics_enabled:
-            deprecate(
-                "Datadog App Analytics is deprecated and will be removed in a future version. "
-                "App Analytics can be enabled via DD_TRACE_ANALYTICS_ENABLED and DD_ANALYTICS_ENABLED "
-                "environment variables and ddtrace.config.analytics_enabled configuration. "
-                "These configurations will also be removed.",
-                category=DDTraceDeprecationWarning,
-            )
+        # DD_ANALYTICS_ENABLED is not longer supported, remove this functionatiy from all integrations in the future
+        self._analytics_enabled = False
```

Comment:
Similar to the changes above, should we log that analytics_enabled is not supported?

Reply:
`2.x` already has the deprecation warning. I think that's enough. 

I think we should just call out the removal in a release note. Setting `DD_TRACE_ANALYTICS_ENABLED` should be a no-op in 3,0.